.github/workflows/publish-release-reusable.yml

```diff
@@ -70,6 +70,32 @@ jobs:
         java-version: 1.8
     - uses: actions/setup-node@v1
     - run: npm install bignumber.js
+
+    - name: Build Dafny
+      run: |
+        dotnet build dafny/Source/Dafny.sln
+        rm dafny/Binaries/*.nupkg
```

Comment:
Is that really necessary?

Reply:
I think it'd actually be okay to remove this whole step and then remove `--no-build` from the later `dotnet pack` commands. However, if we have `dotnet build` then we do, unfortunately, have to remove the `.nupkg` files it creates.